src/MessageContainer.js

```diff
@@ -135,10 +135,20 @@ export default class MessageContainer extends React.Component {
     );
   }
 
+  endReachedProps(){
+    let condProps = {}
+    if (this.props.onTopReached){ condProps.onEndReached = this.props.onTopReached }
```

Comment:
What I'm thinking is that maybe this pull request should be done at the `InvertibleScrollView` so this component will benefit from it as all the others using `InvertibleScrollView`.
what do you think @dgdavid and @kfiroo ?